circuit/src/get_header.nr

```diff
@@ -26,7 +26,7 @@ struct BlockHeaderPartial {
 #[oracle(get_header)]
 unconstrained fn get_header_oracle(_block_no: Field) -> BlockHeaderPartial {}
 
-unconstrained fn get_header(block_no: Field) -> BlockHeaderPartial {
+unconstrained pub(crate) fn get_header(block_no: Field) -> BlockHeaderPartial {
```

Comment:
For a second I thought - that should be just public, but I guess unconstraint functions should be private / crate public by default, so that someone don't call them accidentally.